coursier.bzl

```diff
@@ -119,6 +119,35 @@ def _execute(repository_ctx, cmd, timeout = 600, environment = {}, progress_mess
         quiet = not verbose,
     )
 
+def _execute_with_argsfile(
+        repository_ctx,
+        tool,
+        tool_name,
+        progress_message,
+        error_description,
+        files_to_inspect):
+    # Avoid argument limits by putting list of files to hash into a file
+    repository_ctx.file(
+        "{}_argsfile".format(tool_name),
```

Comment:
This implies that each tool can only ever be used once....

Reply:
Once per repository, yeah. Added a comment :)